pkg/bpf/bpffs.go

```diff
@@ -16,23 +16,27 @@ package bpf
 
 import (
 	"fmt"
-	"os/exec"
+	"os"
 	"path/filepath"
-	"strconv"
-	"strings"
 	"sync"
 	"syscall"
 
 	"github.com/cilium/cilium/pkg/lock"
 	"github.com/cilium/cilium/pkg/logging/logfields"
-	"github.com/cilium/cilium/pkg/pipeexec"
+	"github.com/cilium/cilium/pkg/mountinfo"
+)
 
-	"golang.org/x/sys/unix"
+const (
+	// defaultMapRoot is the default path where BPFFS should be mounted
+	defaultMapRoot = "/sys/fs/bpf"
+	// defaultMapRootFallback is the path which is used when /sys/fs/bpf has
+	// a mount, but with the other filesystem than BPFFS.
+	defaultMapRootFallback = "/run/cilium/bpffs"
```

Comment:
I'm not sure if it makes sense to move these outside this package but, since the last time we merged this PR, we added a pkg/defaults that might also be a good place for constants like this.